### DISCUSSION TITLE
Unraid docker install corrupts Flash USB

### DISCUSSION THREAD

**Discussion by phreaknes** [blakeblackshear/frigate#15366] [General]
UnRaid Server OS version: 6.12.13
Frigate version: 0.14
Motherboard: Supermicro X10SRH-CF

I installed Frigate for the first time in early to mid October 2024 with the minimal settings for 2 camera's. I also have a Coral USB TPU connected at time of install. I did a base config just to get the cameras up on the screen. After about 24 hours after the install I got this message:
![unraid flash error](https://github.com/user-attachments/assets/237da109-07dd-46a6-bd7c-bccb26bdecda)

I spent a lot of time rebuilding / replacing USB drives from backups and trying to diag this problem and started [this thread ](https://forums.unraid.net/topic/178434-second-os-usb-drive-to-fail-in-a-week-still-waiting-on-support-to-contact-me-on-my-first-failure-options-recomendation/#comment-1480683). During this time I was able to figure out that I can do a Windows 10 USB repair and get the system back rather than rebuilding / restoring from backup and creating new Flash drives. 

I opened a ticket with Unraid support and through testing and log reviews they found this.

`Nov  4 20:50:15 PhreakServe kernel: usb 3-6: device descriptor read/8, error -110
Nov  4 20:50:15 PhreakServe kernel: usb 3-6: USB disconnect, device number 3
Nov  4 20:50:15 PhreakServe emhttpd: Unregistered Flash device blacklisted (EBLACKLISTED2)`

I checked the USB buses to make sure they're not shared, just in case. 
![image](https://github.com/user-attachments/assets/be625f56-97dc-4ae0-9b08-03e9195bed8a)

I finally narrowed it down to the Frigate docker after restoring the server, backups, and the flash drive but only spinning up one docker container at a time. I left the frigate docker up and within 24 hours of starting, I would get the USB flash error. I have recreated the issue 3 times, all with the same result. 

As requested by 'yayitazale' on the Unraid support forum for Frigate I started this ticket. There are [several post ](https://forums.unraid.net/topic/98064-support-blakeblackshear-frigate/page/53/#findComment-1483908)that have a little context with some Q&A though out the thread.  

I've been at this for a couple of months and 3 USB drives, I'm at my wits end but I'm invested as I based my config, camera purchases and storage decisions based off of this application. I hope the community can help. 

here is a shot of the running configs for unraid and Frigate. 

![Screenshot 2024-12-05 at 14-45-59 PhreakServe_UpdateContainer](https://github.com/user-attachments/assets/bad47d79-5cbb-4d69-b2be-ebc7968bec4c)
![Screenshot 2024-12-05 at 14-52-56 Config Editor - Frigate](https://github.com/user-attachments/assets/32694bb2-20a3-43bf-9db1-1fdacfbaa44d)

**Comment by NickM-27** [comment]
That is very odd, Unraid is quite popular with Frigate users (including myself) and I have been running unraid & Frigate for 3 years without any issues at this point. 

I'd suggest checking your docker install settings in unraid and see what the docker volume installation type and location is

**Reply by phreaknes** [reply]
First think I checked. It's all the same paths as every other container install.

**Reply by NickM-27** [reply]
the setting I am talking about is not a per container setting

**Reply by phreaknes** [reply]
Ok I'm open to suggestions. What settings should I be looking at and for? Is this an UnRaid setting, Docker setting or Frigate setting. 

> the setting I am talking about is not a per container setting

**Reply by NickM-27** [reply]
in docker settings

![Screen Shot 2024-12-10 at 06 40 35 AM](https://github.com/user-attachments/assets/6cc7c66f-9b37-4193-83a7-26c35e0479e0)

**Reply by phreaknes** [reply]
![image](https://github.com/user-attachments/assets/fd23cc48-cd70-4b05-b3f9-9ecf15f5b220)

**Reply by NickM-27** [reply]
what are the settings for the system share?

**Reply by phreaknes** [reply]
![image](https://github.com/user-attachments/assets/ee9f5f1b-6c5c-4916-a5f0-eaf7a6916ab3)

**Reply by NickM-27** [reply]
weird, doesn't seem that docker is interacting with the USB flash drive at all. Not sure what to suggest at this point, as I said I have been running Unraid with Frigate for 3+ years and never had any issue

**Reply by phreaknes** [reply]
Any other Ideas? I'm to invested in this to not work.
